serverless-infrastructure.yml

```diff
@@ -12,7 +12,6 @@ provider:
       - ${env:SUBNET_ID_A}
       - ${env:SUBNET_ID_B}
       - ${env:SUBNET_ID_C}
-    VpcId: ${env:VPC_ID}
```

Comment:
Why did we remove this?

Reply:
In the logs of the deployment it said it was an invalid config

Reply:
I think it is necessary for GKR, but I'm not sure.